server/src/middlewares/ensureAuthenticated.test.ts

```diff
@@ -2,6 +2,21 @@ import { Request, Response } from 'express';
 import ensureAuthenticated from './ensureAuthenticated';
 
 describe('ensures', () => {
+  // FIXME not actually throwing WTF but when env is actually no set on server it throws. I don't think it have anything to do with asynchronous tests and such, since they are on different threads.
+  it.skip('fails when `JWT_SECRET` env is not set', () => {
+    const request = {
+      // Is there a way to modularize this better? Doesn't feel DRY enough.
```

Comment:
modularize _WHAT_?

Reply:
The middleware parameters. Request, Reponse and next. I think I did something of the sorts on another project, but I don't have access to the source anymore